war/src/main/less/simple-page.less

```diff
@@ -49,6 +49,18 @@ a {
   text-align: center;
 }
 
+.simple-page .safe-restarting {
+  text-align: center;
+  border-color: #009600;
+  border-width: 2px;
+  border-radius: 2px;
+  border-style: solid;
+  width: 50%;
+  margin: 5% auto auto;
+  padding: 5px;
+  background-color: #44ff001f;
```

Comment:
have you tried the colours from the existing colour palette?
https://weekly.ci.jenkins.io/design-library/Colors/

Reply:
![image](https://user-images.githubusercontent.com/41284403/216291492-5f3c08e6-181a-407c-95a3-55df52069ef7.png)
I now use pre-defined colors.